logo.js

```diff
@@ -1600,10 +1738,12 @@ function LogoInterpreter(turtle, stream, savehook)
   def("true", function() { return 1; });
   def("false", function() { return 0; });
 
+// FIXME: needs to handle promises
```

Comment:
So we should probably throw (internal error) if a proc which is a special form returns a non-thenable.